### ISSUE TITLE
Unclear method for retrieving an alias

### ISSUE THREAD

**Issue by TowardsDeath** [zicht/url-bundle#7]
I was looking in the `Aliasing` class for a way to lookup the public alias of an internal URL, but I couldn't find it. It turns out this method is called `hasPublicAlias`, which I initially overlooked because I assumed it returned a boolean.

My proposal: create a method `getPublicAlias` that does exactly the same. `hasPublicAlias` can then be forwarded and marked deprecated. (The same goes for `hasInternalAlias`.)

Happy to work up a PR if this sounds like a good idea.

**Comment by boudewijn-zicht** [comment]
I disagree with the above proposal, I have commented in the associated PR.
